### DISCUSSION TITLE
希望配置项的首页地址可以携带参数

### DISCUSSION THREAD

**Discussion by bujidao** [NervJS/taro#12355] [Q & A] [answered]
### 这个特性解决了什么问题？
项目需求是：通过 公众号携带参数的方式进入小程序首页，比如“/pages/index?userid=112233”

本地开发的时候，为了模拟正式环境中从公众号进入小程序的首页的场景，所以也希望首页地址和实际中的格式一样

因此尝试配置taro的app.config.js/pages[0] = /pages/index?userid=112233，然而编译后，抛出错误“找不到/pages/index?userid=112233.wxml”

### 这个 API 长什么样？
/pages/index?userid=112233

**Comment by yuconora** [comment]
参数应该是在 onLoad 里面获取吧 或者 hooks 里面 使用 
```js
const { params } = useRouter()
```

**Comment by TheKonka** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/22173084/184465442-9bc69753-4bb4-4b95-8c8f-c184b26bef09.png)
参数可以在这里加哦
